client/src/components/users/UserCard.vue

```diff
@@ -4,12 +4,15 @@
   <ion-item
     class="user-card-item ion-no-padding"
     :detail="false"
-    :class="{ selected: user.isSelected, 'no-padding-end': !user.isSelected }"
+    :class="{ selected: user.isSelected, 'no-padding-end': !user.isSelected, revoked: user.isRevoked() }"
     @click="$emit('click', $event, user)"
     @mouseenter="isHovered = true"
     @mouseleave="isHovered = false"
   >
-    <div class="card-checkbox">
+    <div
+      class="user-card-checkbox"
+      v-if="!user.isRevoked()"
+    >
       <!-- eslint-disable vue/no-mutating-props -->
       <ion-checkbox
         aria-label=""
```

Comment:
remove this empty aria-label?

Reply:
No, it's required otherwise we get a warning in the console.